toggle-dummy.js

```diff
@@ -1,8 +1,27 @@
-  $(function(){
-    var button =
-      $('<button type="button" onclick="toggleDummy()"> Show dummies</button>');
-    $("body").prepend(button);
-  });
-  function toggleDummy(){
-    $(".dummy").toggleClass("is-active")
+$(function(){
+  addButton()
+  if (dummyOn()){
+    toggleDummy();
   }
+});
+
+function addButton() {
+  var button = $('<button type="button" onclick="toggleDummy();toggleStorage()"> Show dummies</button>');
+  $("body").prepend(button);
+}
+
+function toggleDummy(){
+  $(".dummy").toggleClass("is-active");
```

Comment:
@drapergeek I did have it in here initially but then realized I wanted to toggleDummy() on document.ready if dummy was on in local storage. So I couldn't add toggleStorage in the same function or else every time a page was reloaded the local storage would toggle as well, which of course is not what we want.

I guess the solution then is to have a third function that the onclick can call that calls the two.